lib/core/src/Cardano/Wallet/DB/Store/Checkpoints.hs

```diff
@@ -584,21 +594,33 @@ instance
             mkSharedState = Shared.SharedState prefix accXPub pTemplate dTemplateM gap
         prologue <- lift $ multisigPoolAbsent wid <&> \case
             True ->  mkSharedState Shared.Pending
-            False -> mkSharedState $ Shared.Active
-                $ Shared.newSharedAddressPool @n gap pTemplate dTemplateM
+            False -> mkSharedState $ Shared.Active $ Shared.SharedAddressPools
+                (Shared.newSharedAddressPool @n @'UtxoExternal gap pTemplate dTemplateM)
+                (Shared.newSharedAddressPool @n @'UtxoInternal gap pTemplate dTemplateM)
+                Shared.emptyPendingIxs
```

Comment:
Nope. 😱 The pending indices need to be loaded with the prologue here; very similar to how `SeqState` does it.